cmd/osbuild-image-tests/main_test.go

```diff
@@ -389,12 +389,16 @@ func testBootUsingAzure(t *testing.T, imagePath string) {
 
 func testBootUsingOpenStack(t *testing.T, imagePath string) {
 	creds, err := openstack.AuthOptionsFromEnv()
+	currentArch := common.CurrentArch()
 
 	// if no credentials are given, fall back to qemu
 	if (creds == gophercloud.AuthOptions{}) {
 		log.Print("No OpenStack credentials given, falling back to booting using qemu")
 		testBootUsingQemu(t, imagePath)
 		return
+		// skip on aarch64 because we don't have aarch64 openstack or kvm machines
+	} else if currentArch == "aarch64" {
```

Comment:
I would move this check above the previous check. If there are no OpenStack creds, qemu boot will still be tried.

Reply:
Right, that wouldn't work locally, I swapped it, thanks!